### PR TITLE
Add RichIterableToImmutableCollectionsTest to Converter Method Kata.

### PR DESCRIPTION
**Pull request by donraab**
Signed-off-by: Donald Raab <Donald.Raab@bnymellon.com>

### CODE REVIEW

File: converter-method-kata-solutions/src/test/java/org/eclipse/collections/convertermethodkata/RichIterableToImmutableCollectionsTest.java [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    public static final Person MARY_SMITH = new Person("Mary", "Smith", 25);
+    public static final Person TED_FIELDS = new Person("Ted", "Fields", 30);
+    public static final Person SALLY_GOLD = new Person("Sally", "Gold", 42);
+    private final ImmutableList<Person> people = Lists.immutable.with(MARY_SMITH, TED_FIELDS, SALLY_GOLD);
+
+    @Test
+    public void toList()
+    {
+        Interval interval = Interval.oneTo(5);
+        // Convert interval to a MutableList<Integer>
```
Comment (prathasirisha) [inline converter-method-kata-solutions/src/test/java/org/eclipse/collections/convertermethodkata/RichIterableToImmutableCollectionsTest.java:58]: Hi @donraab, this comment should say ImmutableList. Same pattern in the rest of the PR.

**Review by prathasirisha — COMMENTED** [review]
Hi @donraab, added one comment. The rest of the changes look good.

File: converter-method-kata-solutions/src/test/java/org/eclipse/collections/convertermethodkata/RichIterableToImmutableCollectionsTest.java (hunk 1 above)
Comment (donraab) [reply converter-method-kata-solutions/src/test/java/org/eclipse/collections/convertermethodkata/RichIterableToImmutableCollectionsTest.java]: Good catch. Thanks!

**Review by prathasirisha — APPROVED** [review]
LGTM.

**Pull request merged**